install_config/aggregate_logging.adoc

```diff
@@ -705,66 +701,51 @@ $ oc edit configmap/logging-elasticsearch
-the pods.
+the pods.  Persisting these changes across deployments can be accomplished by setting
+`openshift_logging_es_allows_cluster_reader` to `true` in the inventory file.
 
 [[aggregated-fluentd]]
 === Fluentd
 
 Fluentd is deployed as a DaemonSet that deploys replicas according to a node
-label selector (which you can specify with the deployer parameter
-`*fluentd-nodeselector*`; the default is `logging-infra-fluentd`).
-
-Once you have ElasticSearch running as desired, label the nodes intended for
-Fluentd deployment to feed their logs into ES. The example below would label a
-node named `node.example.com` using the default Fluentd node selector:
-
-----
-$ oc label node/node.example.com logging-infra-fluentd=true
-----
-
-Alternatively, you can label all nodes with:
-
-----
-$ oc label node --all logging-infra-fluentd=true
-----
-
-[NOTE]
-====
-Labeling nodes requires cluster administrator capability.
-====
+label selector, which you can specify with the inventory parameter
+`openshift_logging_fluentd_nodeselector` and the default is `logging-infra-fluentd`.
+As part of the OpenShift cluster installation, it is recommended that you add the
+Fluentd node selector to the list of persisted
+xref:../install_config/install/advanced_install.adoc#configuring-node-host-labels[node labels].
 
 [[fluentd-use-journald]]
 *Having Fluentd Use the Systemd Journal as the Log Source*
 
 By default, Fluentd reads from *_/var/log/messages_* and
 *_/var/log/containers/<container>.log_* for system logs and container logs,
 respectively. You can instead use the systemd journal as the log source. There
-are three deployer configuration parameters available in the deployer ConfigMap:
+are three inventory parameters available:
 
 [cols="3,7",options="header"]
 |===
 |Parameter
 |Description
 
-| `use-journal`
-|The default is empty, which tells the deployer to have Fluentd check which log
+| `openshift_logging_use_journal`
+|The default is empty, which configures Fluentd to check which log
 driver Docker is using. If Docker is using `--log-driver=journald`, Fluentd
 reads from the systemd journal, otherwise, it assumes docker is using the
 `json-file` log driver and reads from the *_/var/log_* file sources. You can
-specify the `use-journal` option as `true` or `false` to be explicit about
+specify the `openshift_logging_use_journal` option as `true` or `false` to be explicit about
 which log source to use. Using the systemd journal requires `docker-1.10` or
 later, and Docker must be configured to use `--log-driver=journald`.
 
-| `journal-source`
-|The default is empty, so that when using the systemd journal, Fluentd first looks for
-*_/var/log/journal_*, and if that is not available, uses *_/run/log/journal_*
-as the journal source. You can specify `journal-source` with an explicit
-journal path. For example, if you want Fluentd to always read logs
-from the transient in-memory journal, set `journal-source`=*_/run/log/journal_*.
+// | `journal-source`
```

Comment:
@ewolinetz I don't believe we ever added this supporting parameter back in to ansible.  Do we need it?  cc @richm